src/main/kotlin/io/libp2p/core/protocol/Negotiator.kt

```diff
@@ -0,0 +1,135 @@
+package io.libp2p.core.protocol
+
+import io.libp2p.core.events.ProtocolNegotiationFailed
```

Comment:
@raulk you forgot to add those event classes

Reply:
Pushed!